### ISSUE TITLE
eclipse pdt (neon) symfony formatter

### ISSUE THREAD

**Issue by eldiablo62** [pulse00/Symfony-2-Eclipse-Plugin#249]
Hi, 

It is possible to add a file to import for have a good formatter for Symfony standard ? (like PSR-2 for exemple) 

Thanks

**Comment by zulus** [comment]
Yes, it's possible. What should be changed in compare to built-in PSR ?

**Comment by eldiablo62** [comment]
I have found this guide for the standard 
http://symfony.com/doc/current/contributing/code/standards.html
And there is this tools for test 
https://github.com/FriendsOfPHP/PHP-CS-Fixer
php php-cs-fixer.phar fix /path/to/project --level=symfony

I m not an expert that's why if eclipse explain me my error it will be good :)

**Comment by eldiablo62** [comment]
I have found this but i doesnt work when i import it to eclipse 
https://github.com/djoos/Symfony2-coding-standard

**Comment by eldiablo62** [comment]
Fatal error: Uncaught PHP_CodeSniffer_Exception: Referenced sniff PSR1 does not exist in 
when i run phpcs c:\users\toto\.p2\pool\plugins\org.phpsrc.eclipse.pti.library.pear_1.2.2.R20120127000000\php\library\PEAR\PHP\CodeSniffer.php:794
